### PR TITLE
chore: update to alpha 29

### PR DESCRIPTION
**Pull request by PatrickJS**
updated to 29 and everything works

### CODE REVIEW

File: src/app/LoggedInOutlet.ts [hunk 1]
```diff
@@ -26,6 +24,6 @@ export class LoggedInRouterOutlet extends RouterOutlet {
     if (!this.publicRoutes[url] && !localStorage.getItem('jwt')) {
       instruction.component = Login;
     }
-    super.activate(instruction);
+    return super.activate(instruction);
```
Comment (PatrickJS) [inline src/app/LoggedInOutlet.ts:27]: you must `return` here otherwise you will have an error `TypeError: Cannot read property 'then' of undefined`

**Comment by PatrickJS** [comment]
in alpha.30 we can switch to `Http` service (headers doesn't work in 29 atm without replacing XHRBackend)

File: src/home/home.ts [hunk 2]
```diff
@@ -13,18 +13,17 @@ let template = require('./home.html');
   selector: 'home'
 })
 @View({
-  template:`<style>${styles}</style>\n${template}`,
-  directives: [coreDirectives]
+  styles: [ styles ],
```
Comment (PatrickJS) [inline src/home/home.ts:16]: we have `styles` metadata now

File: src/home/home.ts [hunk 3]
```diff
@@ -13,18 +13,17 @@ let template = require('./home.html');
...
 export class Home {
   jwt: string;
   decodedJwt: string;
-  router: Router;
   response: string;
   api: string;
 
-  constructor(router: Router) {
-    this.router = router;
+  constructor(public router: Router) {
```
Comment (PatrickJS) [inline src/home/home.ts:26]: we should use `public` typescript helper

File: src/index.ts [hunk 4]
```diff
@@ -12,6 +12,7 @@ bootstrap(
   App,
   [
     formInjectables,
-    routerInjectables
+    routerInjectables,
+    httpInjectables
```
Comment (PatrickJS) [inline src/index.ts:16]: pending alpha.30 release with Headers fix

File: src/home/home.ts (hunk 2 above)
Comment (mgonto) [inline src/home/home.ts:16]: yey!

File: src/home/home.ts (hunk 3 above)
Comment (mgonto) [inline src/home/home.ts:26]: I wanted to put one and one so that people can learn both.

**Comment by mgonto** [comment]
Thanks :).

Merging!

**Pull request merged**